inspire/modules/deposit/workflows/literature.py

```diff
@@ -506,9 +506,10 @@ def restructure_ffts(fft):
         userid = deposition.user_id
         user = UserInfo(userid)
         email = user.info.get('email', '')
-        external_ids = UserEXT.query.filter_by(id_user=userid).all()
+        external_ids = UserEXT.query.filter_by(id=userid).all()
```

Comment:
mysql> select * from userEXT;
+---------------------+--------+---------+
| id                  | method | id_user |
```
+---------------------+--------+---------+
| 0000-0002-3598-8536 | orcid  |      87 |
+---------------------+--------+---------+
1 row in set (0.00 sec)
```

https://github.com/inveniosoftware/invenio/blob/pu/invenio/modules/accounts/models.py#L325

you can see how the Foreign Key is defined.